docs/source/using_doctr/using_models.rst

```diff
@@ -92,12 +92,12 @@ Detection predictors
 
     import numpy as np
     from doctr.models import detection_predictor
-    predictor = detection_predictor('db_resnet50')
+    model = detection_predictor('db_resnet50')
     dummy_img = (255 * np.random.rand(800, 600, 3)).astype(np.uint8)
     out = model([dummy_img])
 
 You can pass specific boolean arguments to the predictor:
-
+* `pretrained`: if you want to use a model that has been pretrained on a specific dataset, this will load the corresponding weights to enhance the model's performance.
```

Comment:
@khanfarhan10 I think we should drop `to enhance the model's performance ` sounds a bit missleading because otherwise it's random init and will lead to no or useless results ^^ wdyt ?

Reply:
Modified to the following:

`pretrained`: if you want to use a model that has been pretrained on a specific dataset, setting `pretrained=True` this will load the corresponding weights. If `pretrained=False`, which is the default, would otherwise lead to a random initialization and would lead to no/useless results.

Reply:
@felixdittrich92 kindly have a re-look.